.github/workflows/tests.yaml

```diff
@@ -0,0 +1,133 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master, 'maint-*' ]
+    paths:
+      - '.github/workflows/tests.yaml'
+      - 'requirements*.txt'
+      - 'setup.py'
+      - 'setup.cfg'
+      - 'MANIFEST.in'
+      - 'pyproject.toml'
+      - 'scripts/**'
+      - 'fiona/**'
+      - 'tests/**'
+  pull_request:
+    branches: [ master, 'maint-*' ]
+    paths:
+      - '.github/workflows/tests.yaml'
+      - 'requirements*.txt'
+      - 'setup.py'
+      - 'setup.cfg'
+      - 'MANIFEST.in'
+      - 'pyproject.toml'
+      - 'scripts/**'
+      - 'fiona/**'
+      - 'tests/**'
+  schedule:
+    - cron:  '0 0 * * 0'
+
+jobs:
+  docker_tests:
+    runs-on: ubuntu-latest
+    name: Docker | GDAL=${{ matrix.gdal-version }} | python=${{ matrix.python-version }}
+    container: osgeo/gdal:ubuntu-small-${{ matrix.gdal-version }}
```

Comment:
The gdal project recently switched from docker hub to ghcr: https://github.com/OSGeo/gdal/issues/7447

Reply:
Yes, I'm going to make a ticket about that.